packages/kununu-logo/package.json

```diff
@@ -12,6 +12,7 @@
     "react": ">=15.0.0"
   },
   "dependencies": {
+    "kununu-theme": "^3.0.0",
```

Comment:
Here too, this should probably be in the peerDeps. Which means we can just add it to the devDependencies.